<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Allgemein</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="QSpinBox" name="refreshRateSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Plot-Aktualisierungsrate</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Telegram-Bot-Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="telegramNameLineEdit">
            <property name="text">
             <string>RTOC-Remote</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Sprache</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="languageComboBox">
            <item>
             <property name="text">
              <string>English</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Deutsch</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Backup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="backupCheckbox">
            <property name="text">
             <string>Backup aktivieren</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Backup-Intervall</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="backupTimeEdit">
              <property name="date">
               <date>
                <year>2000</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="minimumDate">
               <date>
                <year>2000</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>ss'd' HH'h' mm'min'</string>
              </property>
              <property name="currentSectionIndex">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Backup-Verzeichnis</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="backupDirButton">
              <property name="text">
               <string>Verzeichnis ändern</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="backupIfFullCheckbox">
            <property name="text">
             <string>Auto-Backup wenn voll</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="backupAutoOnCloseCheckbox">
            <property name="text">
             <string>Auto-Backup beim Beenden von RTOC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="backupLoadOnOpenCheckbox">
            <property name="text">
             <string>Backup beim Start automatisch laden</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="backupClearCheckbox">
            <property name="text">
             <string>Daten aus Sitzung leeren nach Backup</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Aufräumen</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Gespeicherte Signal-Stile leeren</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Liste 'Zuletzt bearbeitet' leeren</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="clearSignalStylesButton">
            <property name="text">
             <string>Signal-Stile löschen</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="clearTelegramClientListButton">
            <property name="text">
             <string>Telegram-Verbindungen beenden</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Telegram-Client-Liste leeren</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="clearLastEditedListButton">
            <property name="text">
             <string>'Zuletzt bearbeitet' leeren</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Liste 'Bekannte TCP-Hosts' leeren</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="clearTCPHostListButton">
            <property name="text">
             <string>'Bekannte TCP-Hosts' leeren</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Ansicht</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="6" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Signal-Inactivity-TimeOut</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="signalDropDownComboBox">
            <item>
             <property name="text">
              <string>Icons</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Text</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="openGLCheckBox">
            <property name="text">
             <string>Aus</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>OpenGL</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Signal-Dropdownmenü</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Neue Signale sofort darstellen</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Weave</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="weaveCheckBox">
            <property name="text">
             <string>Aus</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="autoShowCheckBox">
            <property name="text">
             <string>Aus</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Anti-Aliasing</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="antialiasingCheckBox">
            <property name="text">
             <string>Aus</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="darkmodeCheckBox">
            <property name="text">
             <string>Aus</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Fensterposition wiederherstellen</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Dunkle Darstellung</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="restoreWidgetPositionCheckBox">
            <property name="text">
             <string>Aus</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="timeOutSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>PostgreSQL Datenbank</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="postgresqlCheckbox">
            <property name="text">
             <string>PostgreSQL aktivieren</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Nutzername</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="postgresqlUserLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Passwort    </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="postgresqlPasswordLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Host           </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="postgresqlHostLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="postgresqlPortSpinBox">
              <property name="maximum">
               <number>99999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Datenbankname</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="postgresqlNameLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="postgresqlCheckButton">
            <property name="text">
             <string>Verbindung testen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Zurücksetzen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
